test/units/modules/cloud/amazon/test_kinesis_stream.py

```diff
@@ -230,9 +236,21 @@ def test_retention_action_invalid(self):
         )
         self.assertFalse(success)
 
+    def test_update_shard_count(self):
+        client = boto3.client('kinesis', region_name=aws_region)
```

Comment:
I was following the style of the other tests. It should be all changed at once in a separate PR. What do you think?